### PR TITLE
Bonding Price should not be zero

### PR DESCRIPTION
**Pull request by AddressXception**
set the bonding curve price call to at least 1 and add tests

### CODE REVIEW

File: contracts/math/BondingCurve.sol [hunk 1]
```diff
@@ -8,6 +8,7 @@ contract BondingCurve {
 
     // Approximate .001x^2+.000 000 000 000 000 000 000 000 0000999x^{8}
     function curve(uint256 n) public view returns (uint256) {
+        require(n > 0, "BondingCurve: starting position cannot be zero");
```
Comment (CryptoSpaces) [inline contracts/math/BondingCurve.sol:11]: Could we return a start value when N == 0?
or have the bonding curve have a constant

Then we don't need to do the +1 on getPrice

File: contracts/token/RarePizzasBox.sol [hunk 2]
```diff
@@ -51,7 +51,7 @@ contract RarePizzasBox is
 
     // IRarePizzasBox
     function getPrice() public view override returns (uint256) {
-        return super.curve(super.totalSupply());
+        return super.curve(super.totalSupply() + 1);
```
Comment (CryptoSpaces) [inline contracts/token/RarePizzasBox.sol:54]: Would prefer not to go past totalSupply too much

File: test/rarepizzasbox.test.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { expect, use } from "chai";
-import { BigNumber, Contract } from 'ethers';
+import { BigNumber, Contract, utils } from 'ethers';
```
Comment (CryptoSpaces) [inline test/rarepizzasbox.test.ts:2]: What is this utils for?

Comment (AddressXception) [reply test/rarepizzasbox.test.ts:2]: unused removing

File: contracts/math/BondingCurve.sol (hunk 1 above)
Comment (AddressXception) [reply contracts/math/BondingCurve.sol:11]: trying to keep the concerns separate and let the bonding curve be a pure function

File: contracts/token/RarePizzasBox.sol (hunk 2 above)
Comment (AddressXception) [reply contracts/token/RarePizzasBox.sol:54]: similar to the above i think this is fine for now.  this logic is going to change as part of #18

**Review by CryptoSpaces — APPROVED** [review]

**Pull request merged**